configure.py

```diff
@@ -828,7 +828,7 @@ def pkg_config(pkg):
   otherwise (None, None, None, None)"""
   pkg_config = os.environ.get('PKG_CONFIG', 'pkg-config')
   args = []  # Print pkg-config warnings on first round.
-  retval = ()
+  retval = []
```

Comment:
The doc string above this line should be adapted to match the new return type.

Reply:
The docstring is correct as is given line 848.

Reply:
Ah, I missed that change!